types/react-with-styles/lib/ThemedStyleSheet.d.ts

```diff
@@ -1,4 +1,4 @@
-interface ThemedStyleSheet {
+declare const ThemedStyleSheet: {
   registerTheme: (theme: any) => void;
   registerInterface: (registerInterface: any) => void;
   create: (makeFromTheme: any, createWithDirection: any) => any;
```

Comment:
This gets exported as `create: createLTR`
https://github.com/airbnb/react-with-styles/blob/61b02293f7d91cdc5fe9f5d338cc30fc21e4325b/src/ThemedStyleSheet.js#L98

So the type should be `create: (makeFromTheme: any) => any;`